sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -21,6 +22,13 @@
 
 #pragma warning disable SA1402  // File may only contain a single type
 
+[assembly: InternalsVisibleTo("Azure.Storage.DataMovement.Blobs, PublicKey=" +
+    "0024000004800000940000000602000000240000525341310004000001000100d15ddcb2968829" +
+    "5338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc" +
+    "012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265" +
+    "e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593d" +
+    "aa7b11b4")]
+
```

Comment:
TODO: Remove later on after GA and expose APIs used by the DataMovement library and mark APIs as protected.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/32720